cometmock/abci_client/client.go

```diff
@@ -317,6 +317,31 @@ func (a *AbciClient) callClientWithTimeout(client AbciCounterpartyClient, f func
 	}
 }
 
+func (a *AbciClient) SendABCIInfo() (*abcitypes.ResponseInfo, error) {
+	if verbose {
+		a.Logger.Info("Sending Info to clients")
+	}
+	// send Info to all clients and collect the responses
+	f := func(client AbciCounterpartyClient) (interface{}, error) {
+		return client.Client.InfoSync(abcitypes.RequestInfo{})
+	}
+	responses, err := a.callClientsWithTimeout(f, 500*time.Millisecond)
+	if err != nil {
+		return nil, err
+	}
+
+	if a.ErrorOnUnequalResponses {
```

Comment:
Since CometMock is one single component, is it possible to get different responses?

Reply:
Yes - this is calling the many applications that CometMock talks to, so if they give out different responses, we typically want to know about this (since it means the app is nondeterministic)